src/transformers/models/character_bert/configuration_character_bert.py

```diff
@@ -0,0 +1,160 @@
+        num_attention_heads (:obj:`int`, `optional`, defaults to 12):
+            Number of attention heads for each attention layer in the Transformer encoder.
+        intermediate_size (:obj:`int`, `optional`, defaults to 3072):
+            Dimensionality of the "intermediate" (i.e., feed-forward) layer in the Transformer encoder.
+        hidden_act (:obj:`str` or :obj:`function`, `optional`, defaults to :obj:`"gelu"`):
+            The non-linear activation function (function or string) in the encoder and pooler.
+            If string, :obj:`"gelu"`, :obj:`"relu"`, :obj:`"selu"` and :obj:`"gelu_new"` are supported.
+        hidden_dropout_prob (:obj:`float`, `optional`, defaults to 0.1):
+            The dropout probabilitiy for all fully connected layers in the embeddings, encoder, and pooler.
+        attention_probs_dropout_prob (:obj:`float`, `optional`, defaults to 0.1):
+            The dropout ratio for the attention probabilities.
+        max_position_embeddings (:obj:`int`, `optional`, defaults to 512):
+            The maximum sequence length that this model might ever be used with.
+            Typically set this to something large just in case (e.g., 512 or 1024 or 2048).
+        type_vocab_size (:obj:`int`, `optional`, defaults to 2):
+            The vocabulary size of the :obj:`token_type_ids` passed when calling :class:`~transformers.CharacterBertModel` or
+            :class:`~transformers.TFCharacterBertModel`.
+        mlm_vocab_size (:obj:`int`, `optional`, defaults to 100000):
+            Size of the output vocabulary for MLM.
+        initializer_range (:obj:`float`, `optional`, defaults to 0.02):
+            The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
+        layer_norm_eps (:obj:`float`, `optional`, defaults to 1e-12):
+            The epsilon used by the layer normalization layers.
+        use_cache (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether or not the model should return the last key/values attentions (not used by all models). Only
+            relevant if ``config.is_decoder=True``.
+        Example::
+
+        >>> from transformers import CharacterBertModel, CharacterBertConfig
+
+        >>> # Initializing a CharacterBERT helboukkouri/character-bert style configuration
+        >>> configuration = CharacterBertConfig()
+
+        >>> # Initializing a model from the helboukkouri/character-bert style configuration
+        >>> model = CharacterBertModel(configuration)
+
+        >>> # Accessing the model configuration
+        >>> configuration = model.config
+    """
+    model_type = "character_bert"
+
+    def __init__(
+        self,
+        character_embeddings_dim=16,
+        cnn_activation="relu",
+        cnn_filters=[
+            [1, 32],
+            [2, 32],
+            [3, 64],
+            [4, 128],
+            [5, 256],
+            [6, 512],
+            [7, 1024]
+        ],
```

Comment:
Having a list as a default kwarg is dangerous in Python. This should be set to None here, and then update in the init with 
```python
if cnn_filters is None:
    cnn_filters = ...
```